src/hooks/useIAP.ts

```diff
@@ -34,6 +34,7 @@ type IAP_STATUS = {
   getSubscriptions: (skus: string[]) => Promise<void>;
   requestPurchase: typeof iapRequestPurchase;
   requestSubscription: typeof iapRequestSubscription;
+  clearTransactionIOS: () => Promise<void>
```

Comment:
Can you move this under the `finishTransaction` method to group by "logic" methods, and you can import it straight away from this file as 

```ts
import {clearTransactionIOS as iapClearTransactionIOS} from './iap'
```